jobsub/examples/pALPIDEfs/steering-templates_pALPIDEfs_7/hotpixel-tmp.xml

```diff
@@ -26,7 +26,7 @@
    <global>
       <parameter name="LCIOInputFiles"> @LcioPath@/@FilePrefix@-converter.slcio </parameter>
       <parameter name="GearXMLFile" value="@GearFilePath@/@GearFile@"/>
-      <parameter name="MaxRecordNumber" value="@nEventsForHotpixel@"/>
+      <parameter name="MaxRecordNumber" value="@MaxRecordNumber@"/>
```

Comment:
Yes, I saw that also. Sorry, I thought MaxRecordNumber is a new variable and that's why I got confused where it is set. I see now it used to be there in the config file before the changes as well.